front-app/src/api/usersApi.tsx

```diff
@@ -0,0 +1,19 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+require('dotenv').config();
```

Comment:
Don't think we need this, check the vite docs, it should be doing automatically

https://vitejs.dev/guide/env-and-mode.html

<img width="711" alt="image" src="https://user-images.githubusercontent.com/62408261/228853535-73a2c950-5215-48d2-bb8c-87d211e9913b.png">